src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    private LocalDate time;
+
+    /***
+     * Constructor of newly created deadline
+     * @param name
+     * @param time
+     */
+    public Deadline(String name, LocalDate time) {
+        super(name);
+        done = false;
+        this.time = time;
+    }
+
+    /***
+     * Constructor of newly read deadline
+     * @param name
+     * @param time
+     * @param done
+     */
+    public Deadline(String name, LocalDate time, boolean done) {
+        super(name);
+        this.done = done;
+        this.time = time;
+    }
+
+
+    /***
+     * Specify print format
+     * @return print format
+     */
+    public String toString() {
+        if (done) {
+            return ("[D][v] " + name + " | by: "+time.format(DateTimeFormatter.ofPattern("MMM d yyyy")) + "\n");
+        }
```

Comment:
Remember to leave a space between the description and params/return! Include a full stop after the description as well.